requirements.txt

```diff
@@ -13,4 +13,4 @@ django-tastypie>=0.14.0
 beautifulsoup4
 requests_cache
 
--e git+https://github.com/mysociety/sayit#egg=django-sayit
+-e git+https://github.com/mysociety/sayit@testing-updates#egg=django-sayit
```

Comment:
We may want to merge https://github.com/mysociety/sayit/pull/530 and ditch this change?

Reply:
Yep!